homeassistant/components/cover/mqtt.py

```diff
@@ -44,11 +44,17 @@
     vol.Optional(CONF_STATE_CLOSED, default=STATE_CLOSED): cv.string,
     vol.Optional(CONF_OPTIMISTIC, default=DEFAULT_OPTIMISTIC): cv.boolean,
     vol.Optional(CONF_RETAIN, default=DEFAULT_RETAIN): cv.boolean,
+    vol.Optional(CONF_VALUE_TEMPLATE): cv.template,
```

Comment:
Ah yes. We should get rid of those. We should always depend on the `PLATFORM_SCHEMA` from the component we are a platform off. So `light.mqtt` should extend `light.PLATFORM_SCHEMA` etc.